sirepo/package_data/static/js/raydata.js

```diff
@@ -305,10 +305,12 @@ SIREPO.app.directive('scansTable', function() {
         template: `
             <div data-show-loading-and-error="" data-model-key="scans">
               <div>
-                <button class="btn btn-info btn-xs" data-ng-click="addColumn()" style="float: right;"><span class="glyphicon glyphicon-plus"></span></button>
+                <button class="btn btn-info btn-xs" data-ng-click="addColumn()" style="float: right; margin:5px;"><span class="glyphicon glyphicon-plus"></span></button>
+                <button class="btn btn-info btn-xs"  data-ng-show="showPdfColumn" data-ng-click="downloadSelectedAnalyses()" style="float: right; margin:5px;">Download Selected Analysis PDFs</button>
```

Comment:
I think this should only become visible when there are pdfs selected. The UI is a little confusing now where you can select all pdfs and hit this button but nothing happens.

![Screen Shot 2023-03-14 at 4 41 04 PM](https://user-images.githubusercontent.com/10264515/225161932-8b771e1b-8418-4460-b4aa-11a23d04b0fa.png)